docs-md/concepts/collections/overview.md

```diff
@@ -1,7 +1,35 @@
-Although an individual event represents vitally important information, in practice they are almost more useful when grouped together with related events. ksqlDB supports durable collections of events that can be replicated across multiple servers. This means that, similar to replicated Postgres, that if there is a recoverable failure of some of the servers, your events remain permanently stored. It is able to do this by storing data in Kafka topics.
+---
+layout: page
+title: Collections
+tagline:  Streams and tables in ksqlDB
+description: Learn about durable collections of events in ksqlDB
+keywords: ksqlDB, events, collection, stream, table
+---
 
-Collections are sharded, and can arbitrarily grow in size by adding more Kafka brokers to a ksqlDB cluster. This sharding is more commonly known as “partitioning” in Kafka, and is the term that ksqlDB uses as well. Events with the same key are stored on the same partition.
+Although an individual [event](../events.md) represents vitally important
+information, events are more useful when grouped together with related events.
```

Comment:
"when *stored* together with related events" conveys the same thing? Since we talk replication below, this might be a better phrasing IMO

Reply:
Yeah, that sounds better.